tools/log4j2.xml

```diff
@@ -69,6 +69,19 @@
             </Policies>
         </RollingFile>
 
+        <RollingFile name="NRTM4LOG"
+                     append="true"
+                     filePattern="var/log/nrtm4/nrtmlog.%d{yyyyMMdd}"
+                     ignoreExceptions="false">
+            <ThresholdFilter level="debug"/>
+            <PatternLayout>
+                <Pattern>%d{yyyyMMdd HH:mm:ss} %m%n</Pattern>
+            </PatternLayout>
+            <Policies>
```

Comment:
I think one of these log4j2.cml is not being used. Can we remove one that is nit used for tests? We manually copy the log4 xml from db-operational to deployment directory